contentcuration/contentcuration/frontend/shared/views/StudioPage.vue

```diff
@@ -31,10 +34,19 @@
         maxWidth: windowIsLarge.value ? '1000px' : '100%',
       }));
 
-      return { innerStyle };
+      const outerStyle = computed(() => {
+        const marginTop = props.marginTop !== null ? props.marginTop : 104;
+        return {
+          marginTop: `${marginTop}px`,
+          height: `calc(100vh - ${marginTop}px)`,
+        };
+      });
+
+      return { innerStyle, outerStyle };
     },
     props: {
       offline: { type: Boolean, default: false },
+      marginTop: { type: Number, default: null },
```

Comment:
```suggestion
      marginTop: { type: Number, default: 104 },
```

Then you could remove `const marginTop = props.marginTop !== null ? props.marginTop : 104;` and just rely on the default.

Reply:
I changed the default to 104 and simplified the computed property to always use the marginTop value directly